client/app/scripts/superdesk-desks/desks.js

```diff
@@ -642,6 +642,15 @@
                     },
 
                     /**
+                     * Fetch current user destination desks and stage
+                     */
+                    fetchLastDestination: function() {
```

Comment:
Ideally this should be in `preferenceService`. And I think preferenceService already takes care of this. If we use preferenceService we might not need this function.